<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wifi Monitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>611</width>
      <height>341</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Monitor</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>381</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Connection settings</string>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>191</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>Server UDP</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>IP Addr:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_IP_server">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>113</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>192.168.1.35</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_port_server">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>61</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>5000</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_en_server">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>80</y>
          <width>111</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>191</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>Client UDP</string>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>IP Addr:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_ip_client">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>113</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>192.168.1.49</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_port_client">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>61</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>5001</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_en_client">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>80</y>
          <width>111</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>159</y>
        <width>241</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Orientation</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Roll:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_roll">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>71</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_pitch">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>71</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Pitch</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_yaw">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>80</y>
         <width>71</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Yaw</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3d_model">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>3D Model...</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_chart_orientation">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>180</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Chart...</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>160</y>
        <width>141</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Motor Speed</string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Velo A:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_velocA">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>71</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_velocB">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>71</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Velo B:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_veloc_avg">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>71</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Average:</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>160</y>
        <width>191</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Controller </string>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Analog 1:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_analog1_horiz">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>51</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_analog1_vertical">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>51</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Analog 2:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_analog2_vertical">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>50</y>
         <width>51</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_analog2_horiz">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>51</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>PID</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>301</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Angle Control</string>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Kp Cons:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Ki Cons:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Kd Cons:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>120</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Setpoint:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_angle_kp">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>69</width>
         <height>27</height>
        </rect>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_angle_ki">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>69</width>
         <height>27</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_angle_kd">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>69</width>
         <height>27</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_angle_setpoint">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>110</y>
         <width>69</width>
         <height>27</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_angle_zero">
       <property name="geometry">
        <rect>
         <x>78</x>
         <y>110</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Zero</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_angle_set">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Kp Aggr:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>60</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Ki Aggr:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_angle_kd_Aggr">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>80</y>
         <width>69</width>
         <height>27</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_angle_kp_Aggr">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>20</y>
         <width>69</width>
         <height>27</height>
        </rect>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_angle_ki_Aggr">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>50</y>
         <width>69</width>
         <height>27</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>90</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Kd Aggr:</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>151</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Speed Control</string>
      </property>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Kp:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Ki:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Kd:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_speed_kp">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>69</width>
         <height>27</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_speed_ki">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>69</width>
         <height>27</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_speed_kd">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>69</width>
         <height>27</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_speed_zero">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Zero</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_speed_set">
       <property name="geometry">
        <rect>
         <x>2</x>
         <y>110</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>121</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Chart</string>
      </property>
      <widget class="QCheckBox" name="checkBox_angle_out">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Angle output</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_speed_out">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Speed output</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_chart_pid">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Plot</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Control</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>171</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Body</string>
      </property>
      <widget class="QDial" name="dial">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>50</width>
         <height>64</height>
        </rect>
       </property>
      </widget>
      <widget class="QSlider" name="verticalSlider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>29</width>
         <height>160</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_11">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>171</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Head</string>
      </property>
      <widget class="QSlider" name="verticalSlider_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>29</width>
         <height>160</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QSlider" name="verticalSlider_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>29</width>
         <height>160</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
